diff '--color=auto' -ur a/myocamlbuild.ml b/myocamlbuild.ml
--- a/myocamlbuild.ml	2025-04-07 10:07:10.000000000 +0000
+++ b/myocamlbuild.ml	2025-04-07 10:28:41.000000000 +0000
@@ -20,19 +20,22 @@
 
        List.iter begin fun (lib,dir) ->
          flag ["ocaml"; "link"; "c_use_"^lib; "byte"]
-         (S[A"-custom"; A"-cclib"; A"-lstdc++";
+         (S[A"-cc"; A"g++";
+	    A"-custom"; A"-cclib"; A"-lstdc++";
             A"-cclib"; A"-lz";
             A"-ccopt"; A("-L"^lib); 
             A"-cclib"; A("-l"^lib)]);
 
          flag ["ocaml"; "link"; "c_use_"^lib; "native"]
-         (S[A"-cclib"; A"-lstdc++";
+         (S[A"-cc"; A"g++";
+	    A"-cclib"; A"-lstdc++";
             A"-cclib"; A"-lz"; 
             A"-ccopt"; A("-L"^lib); 
             A"-cclib"; A("-l"^lib)]);
 
          flag [ "byte"; "library"; "link" ]
-         (S[A"-cclib"; A"-lstdc++";
+         (S[A"-cc"; A"g++";
+	    A"-cclib"; A"-lstdc++";
             A"-cclib"; A"-lz";
             A"-cclib"; A("-l"^lib);
             A"-dllib"; A("-l"^lib^"_stubs"); 
@@ -41,7 +44,8 @@
          ]);
 
          flag [ "native"; "library"; "link" ]
-         (S[A"-cclib"; A"-lstdc++";
+         (S[A"-cc"; A"g++";
+	    A"-cclib"; A"-lstdc++";
             A"-cclib"; A"-lz";
             A"-cclib"; A("-l"^lib); 
             A"-cclib"; A("-l"^lib^"_stubs")]);
